src/content/blog/whats-new-december-2023.mdx

```diff
@@ -0,0 +1,162 @@
+---
+title: "What's new in Astro - December 2023"
+description: "December: last month of the year, a time to reflect on all the amazing things we've shipped, including Astro 4.0!"
+publishDate: "December 31, 2023"
+authors:
+  - elian
+coverImage: "/src/content/blog/_images/whats-new-shared.webp"
+socialImage: "/src/content/blog/_images/whats-new-december-2023/og.webp"
+lang: "en"
+---
+
+## Releases
+
+Just in time for the end of the year, we released Astro 4.0!
+
+- [Astro 4.0](https://github.com/withastro/astro/releases/tag/astro%404.0.0)
+- [Starlight 0.15](https://github.com/withastro/starlight/releases/tag/%40astrojs%2Fstarlight%400.15.0)
+
+## Docs
+
+The Astro Documentation is now running on Starlight!
+
+We have also officially launched [Astro Docs Docs (AD²)](https://contribute.docs.astro.build) to help you contribute to docs! You'll find all kinds of information whether you're making your first open-source PR or a seasoned member of Team Docs.
+
+Our weekly Discord call “Talking and Doc’ing” is a chance for you to watch Team Docs work in public, jump in a codespace, and contribute to the docs. This month’s topics included:
+
+- Building a new header component for our integrations pages
+- Establishing a community Glossary project
+- Discussing how to write/standardize Changelog messages
+- Adding a mini-showcase with links to themes using each CMS on our CMS guide pages
+- Updating our integration docs to include MDX components and Expressive Code
+
+## Community
+
+### Community Awards November
+
+- 🌟 $250 awarded to Atharva for significant contributions to Astro Docs and active community involvement.
+- 🥳 $250 awarded to McFlyPartages for translating a significant portion of the Astro Docs into French.
+- 🎉 $250 awarded to Hippo for exceptional work releasing, maintaining, and providing support for Expressive Code.
+- 🍾 $250 awarded to Eva Decker for actively supporting the Astro Community on Discord.
+- 🎊 $250 awarded to Jumper for being a super positive, welcoming and supportive presence in the Astro Discord.
+- 🎁 $250 awarded to Happydev for significant work on the Astro Core code base.
+
+Check them out or learn more about the Community Awards on [our community website](https://community.astro.build).
+
+## Showcase
+
+### Content
+
+🚉 [Nuxt 2 to Astro 3 Replatforming – from Setup to Production](https://stevenwoodson.com/blog/replatforming-from-nuxtjs-2-to-astro/) by @stevenwoodson
+
+🌴 [Quick-and-useful: A DIY, self-hosted Linktree and Linkinbio clone with Astro and TinaCMS](https://darko.io/posts/linktree-and-linkinbio-clone-with-astro-and-tinacms/) by @\_seiryuu\_
+
+⚠️ [You are using Astro wrong](https://arshx.substack.com/p/you-are-using-astro-wrong-a-survival) by @arshxcx
```

Comment:
```suggestion
⚠️ Get some tips, because maybe [You are using Astro wrong](https://arshx.substack.com/p/you-are-using-astro-wrong-a-survival) by @arshxcx
```